core/src/datapoint_source/bitpanda.rs

```diff
@@ -33,6 +35,35 @@ pub async fn get_kgau_usd() -> Result<AssetsExchangeRate<KgAu, Usd>, DataPointSo
     }
 }
 
+// Currently only used for testing
```

Comment:
Why use it only for testing? I mean, I don't see why it cannot be used along with other sources in production.

Reply:
If your concern is that it's not a direct ERG/BTC rate, I think it's ok to use it through ERG/USD rate since there are not so many sources for ERG/X so far.

Reply:
The coincap source does find ERG/BTC through combining ERG/USD and USD/BTC. I could not find an ERG/USD source for bitpanda however, and was concerned that sourcing ERG/USD data and USD/BTC data from different sources and combining them could lead to deviation

Reply:
Hm, interesting point. I'm feeling quite the opposite, that combining different sources would make the result more resilient. We have to live with different sources giving slightly different results. I believe that averaging them should be good enough to produce usable data. And in this case it's also kind of averaging but on a lower level.

Reply:
Added a bitpanda+coincap source which does seem to have acceptable deviation levels